extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -13,12 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import {
+  AMP_STORY_PLAYER_EVENT,
+  CONTROL_EVENTS,
+} from '../../../src/amp-story-player/amp-story-player-impl';
```

Comment:
Can you check manually that it only imports what you want in the final amp-story.js build, and no extra code? @mszylkowski has found some cases where it was importing way too much in the past

Reply:
I also compared the sized of the compiled `amp-story-1.0.js` and it's 418kB (new) vs. 417kB (old).